app/views/casework/crime_applications/sections/_properties.html.erb

```diff
@@ -2,7 +2,7 @@
   <h2 class="govuk-heading-l"><%= label_text(:assets) %></h2>
 <% end  %>
 
-<% if capital_details.has_no_properties == 'yes'  %>
+<% if capital_details.properties.empty?  %>
```

Comment:
I just realised when there is a partial capital assessment, this section will show when it should not 🥲. I think we may need to introduce a helper method that can determine if the full means assessment has been undertaken? Happy to discuss

Reply:
If properties are not relevant to the application at submission, they should not be included in the data that is sent to the datastore. Happy to help work out how to stop that form happening.